keps/sig-node/3675-resource-plugin-manager/README.md

```diff
@@ -546,9 +548,20 @@ following interface to manage resource sets.
         +RemoveResource(container)
         +AvailablResources(): resourceset
 
-3. Integration The Container Compute Interface “CCI” inside None and Static CPU Manager policies<br>
+3. Integration the Container Compute Interface “CCI” inside Cpu Manager Policies<br>
 The synchronization of cpuset state is done by a new policy component inside the 
-cpu manager which will be integrated inside the None and Static policies. For simplicity in alpha version we plan to support best-effort QoS for all Pods not requiring a resource management driver (equivalent to none policy). In later phases we plan to support Pods requiring the static policy without the need of resource driver (goal for Beta). The component acts as a bridge between CCI drivers and CPU manager. The drivers provide a decision (resource set stored in CCI Store) about a certain allocation and and cpu manager applies the given cpuset. Similar appoach could be followed with memory management.
+cpu manager which will cover both behaviours: none and static behaviour. The component acts also as
+a bridge between CCI drivers and CPU manager. The drivers provide a decision (resource set stored
```

Comment:
I'll comment about this in more detail later (need to review the full text not just this diff) but just at first glance I have few concerns already here

1. synchronization between two components, especially about the kubelet and this optional component, is always a trick subject. I'm worried this can be hard to get right. Can we list alternatives here and explain why were discarded?
2. shall the synchronization mechanism be reimplemented (and tested!) for each CCI CPU driver or just needs to be done once, possibly by the framework authors?
3. even if the synchronization is rock solid, just the need to synchronize state with cpumanager means that we are bounded and limited by the internal state representation of the cpumanager. IOW, this boils down to two sub-questions: a. do we think all the use case listed previously can be implemented using the current state representation? b. if not, how/where is the extra data needed by CCI stored?

Reply:
We'll delay the synchronization to Beta; get only cpu in this alpha from @klueska  suggestions.